package.json

```
@@ -166,7 +166,7 @@
     "gh-pages": "^3.0.0",
     "glob": "^7.1.1",
     "jest": "^26.6.3",
-    "postcss": "8.3.11",
+    "postcss": "8.4.6",
```

Nit: If we're updating the version then might as well undo locking down the version:
```suggestion
    "postcss": "^8.4.6",
```